UI/WebServerResources/SOGoMailDataSource.js

```diff
@@ -82,9 +82,35 @@ SOGoMailDataSource = Class.create({
         },
 
         _loadCallback: function(http) {
+            var data,
+            re, reMatches, formLine;
             if (http.status == 200) {
                 if (http.responseText.length > 0) {
-                    var data = http.responseText.evalJSON(true);
+                    try {
+                        data = http.responseText.evalJSON(true);
+                    } catch(e) {
```

Comment:
imho it would be much better to check content-type header of the XMLHttpRequest object (if is not json, avoid evalJSON call)